protocols/identify/src/behaviour.rs

```diff
@@ -110,6 +116,7 @@ impl Config {
             interval: Duration::from_secs(5 * 60),
             push_listen_addr_updates: false,
             cache_size: 100,
+            hide_listen_addrs: false,
```

Comment:
I chose for not changing the current behavior by default. I don't know how much impact it would have on various libp2p users.
Otherwise we'd have to warn for potentially breaking some implementations?

Reply:
Sounds good to me.